src/app/zap-templates/zcl/data-model/chip/media-playback-cluster.xml

```diff
@@ -26,134 +26,84 @@ limitations under the License.
     <!-- TODO: Add feature map once it is supported -->
     <description>This cluster provides an interface for controlling Media Playback (PLAY, PAUSE, etc) on a media device such as a TV or Speaker.</description>
 
-    <attribute side="server" code="0x0000" define="MEDIA_PLAYBACK_STATE" type="ENUM8" min="0x00" max="0x02" writable="false" default="0x00" optional="true">playback state</attribute>
+    <attribute side="server" code="0x0000" define="MEDIA_PLAYBACK_STATE" type="PlaybackStateEnum" min="0x00" max="0x02" writable="false" default="0x00" optional="true">playback state</attribute>
     <attribute side="server" code="0x0001" define="MEDIA_PLAYBACK_START_TIME" type="INT64U" min="0x00"  writable="false" default="0xFF" optional="true">start time</attribute>
     <attribute side="server" code="0x0002" define="MEDIA_PLAYBACK_DURATION" type="INT64U" min="0x00"  writable="false" optional="true">duration</attribute>
-    <attribute side="server" code="0x0003" define="MEDIA_PLAYBACK_PLAYBACK_POSITION" type="MediaPlaybackPosition" writable="true" optional="false">position</attribute>
+    <attribute side="server" code="0x0003" define="MEDIA_PLAYBACK_PLAYBACK_POSITION" type="PlaybackPosition" writable="true" optional="false">position</attribute>
     <attribute side="server" code="0x0004" define="MEDIA_PLAYBACK_PLAYBACK_SPEED" type="single" min="0x00"  writable="false" optional="true">playback speed</attribute>
     <attribute side="server" code="0x0005" define="MEDIA_PLAYBACK_PLAYBACK_SEEK_RANGE_END" type="INT64U" min="0x00"  writable="false" optional="true">seek range end</attribute>
     <attribute side="server" code="0x0006" define="MEDIA_PLAYBACK_PLAYBACK_SEEK_RANGE_START" type="INT64U" min="0x00"  writable="false" optional="true">seek range start</attribute>
 
 
-    <command source="client" code="0x00" name="MediaPlay" optional="false">
+    <command source="client" code="0x00" name="PlayRequest" response="PlaybackResponse" optional="false">
       <description>Upon receipt, this SHALL play media.</description>
     </command>
 
-    <command source="client" code="0x01" name="MediaPause" optional="false">
+    <command source="client" code="0x01" name="PauseRequest" response="PlaybackResponse" optional="false">
       <description>Upon receipt, this SHALL pause media.</description>
     </command>
 
-    <command source="client" code="0x02" name="MediaStop" optional="false">
+    <command source="client" code="0x02" name="StopRequest" response="PlaybackResponse" optional="false">
       <description>Upon receipt, this SHALL stop media. User experience is context-specific. This will often navigate the user back to the location where media was originally launched.</description>
     </command>
 
-    <command source="client" code="0x03" name="MediaStartOver" optional="false">
+    <command source="client" code="0x03" name="StartOverRequest" response="PlaybackResponse" optional="false">
```

Comment:
Should be optional="true", as should various other commands here.  Followup....

Reply:
Added to the issue list: https://github.com/project-chip/connectedhomeip/issues/13214